src/index.html

```diff
@@ -1,14 +1,74 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>Moyo header</title>
-    <link rel="stylesheet" href="./style.css">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Moyo Header</title>
+
+    <link
+     rel="preconnect"
+     href="https://fonts.googleapis.com"
+    >
+
+    <link
+     rel="preconnect"
+     href="https://fonts.gstatic.com"
+     crossorigin="anonymous"
+    >
+
+    <link
+     href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+     rel="stylesheet"
+    >
+
+    <link
+     rel="stylesheet"
+     href="style.css"
+    >
+
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#" class="logo-img">
+        <img src="./images/logo.png" alt="Moyo">
+      </a>
+      
+      <nav class="nav">
+        <ul class="nav-list">
+          <li class="nav-item">
+            <a href="#Apple" class="nav-link is-active">Apple</a>
+          </li>
+
+          <li class="nav-item">
+            <a href="#Samsung" class="nav-link">Samsung</a>
+          </li>
+
+          <li class="nav-item">
+            <a href="#Smartphones" class="nav-link">Smartphones</a>
+          </li>
+
+          <li class="nav-item">
+            <a href="#Laptops & Computers" class="nav-link"
+            data-qa="hover">Laptops & Computers</a>
```

Comment:
```suggestion
            <a
              href="#Laptops & Computers"
              class="nav-link"
              data-qa="hover"
            >Laptops & Computers</a>
```

Reply:
move every attribute to new line with +2 indentation if you start doing it
do it if you have more than 2 attributes